<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCanny</class>
 <widget class="QDialog" name="DialogCanny">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>240</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>240</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>141</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="act">
    <item>
     <widget class="QLabel" name="labelApertureSize_3">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Threthold 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="threshold1">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelApertureSize_2">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Threthold 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="threshold2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelApertureSize_4">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Aperture size</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="apertureSlider">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>3</number>
      </property>
      <property name="singleStep">
       <number>2</number>
      </property>
      <property name="sliderPosition">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>71</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLCDNumber" name="lcdNumberThreshold1"/>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcdNumberThreshold2"/>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>30</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcdNumberApertureSize">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="intValue" stdset="0">
       <number>1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>280</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>threshold1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberThreshold1</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>threshold2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberThreshold2</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>apertureSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberApertureSize</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
